My TOP calculator project

mudimurtala opened the pull request:
I finished it recently and I'm ready for review.

mudimurtala committed:
Add html boilerplate code

mudimurtala committed:
Add styles.css file

mudimurtala committed:
Add script.js file

mudimurtala committed:
Add a div element

mudimurtala committed:
Style the .css file

mudimurtala committed:
Add Math functions and operation variables

mudimurtala committed:
Add operator function

mudimurtala committed:
Add button elements

mudimurtala committed:
Update the styling of my elements

mudimurtala committed:
Add functions that handle calculations

FlatBallFlyer reviewed (changes requested):
There is a bug in your code. Follow these directions
```
5 + 5 + 3 =  gives 13, then after using the = you do any more math it fails, as in
- 3 + 4 / 2 = yeilds 342 when it should yield 7
```

mudimurtala committed:
Modify `calculate()` function to fix bug

mudimurtala committed:
Fix a bub in the `handleOperator() function`

mudimurtala committed:
Modify

mudimurtala committed:
Handling how my  works to fix my bug

mudimurtala commented:
I have modified these two functions `handleOperator(op)` and `calculate()` and was able to fix the bug with chaining my operations. However I noticed another little bug again, I'm working on fixing that too.

mudimurtala committed:
Review the functions to fix a bug

mudimurtala committed:
Add variables and remove some to make the calculator work as expected

mudimurtala committed:
I construct a detailed README.md file content

mudimurtala committed:
I construct a detailed README.md file content

mudimurtala commented:
Here’s what I’ve done:
1. I removed the flags: This simplifies my code and has made it more readable and efficient.
2. I created a global `result` variable: It allows the variable to be accessible across different functions, improving the code’s flexibility.
3. I fixed a bug in the `handleNumber` function: I initially used three equal signs (`===`) when comparing the result variable instead of two equal signs (`==`). It took me time to realize this mistake, but I later spotted it.
4. I utilized string methods: I directly assigned values to my `secondNumber` variable using string methods.

I couldn't implement the array/object way you were trying to teach me during our last meeting, but I'll see if I can try it again.
The calculator is now working well to some extent, and I’m looking forward to your review Sir. Thank you!

mudimurtala committed:
Review handleOperator() function so that there is a space before and after an operator when rendered on the display screen

mudimurtala committed:
Add white-space property to .display css class

mudimurtala committed:
Add "result = null" to clear() function to handle possible bug
